packages/ic-asset-certification/src/asset_config.rs

```diff
@@ -217,6 +234,37 @@ pub enum AssetConfig {
         /// the alias `/`, a request for `/` will be served the
         /// asset at `index.html`.
         aliased_by: Vec<String>,
+
+        /// A list of encodings to serve the asset with. Each listing includes
+        /// the encoding of an asset, and the file extension for the encoded
+        /// asset. The router will search for an asset with the provided file
+        /// extension and certify all matching encoded assets, if found.
+        ///
+        /// A list of alternative encodings that can be used to serve the asset.
+        ///
+        /// Each entry is a tuple of the [encoding name](AssetEncoding) and the
+        /// file extension used in the file path. For example, to include Brotli
+        /// and Gzip encodings:
+        /// `vec![(AssetEncoding::Brotli, "br".to_string()), (AssetEncoding::Gzip, "gz".to_string())]`
```

Comment:
These lines are repeated on purpose because there is the same property repeated on two different variants.

```rust
pub enum AssetConfig {
  File {
    // ...
    encodings: Vec<(AssetEncoding, String)>,
  },
  Pattern {
    // ...
    encodings: Vec<(AssetEncoding, String)>,
  }
}
```

It's not ideal to duplicate it, but I think it's nice for developers to have the same information in both places so they don't have to look at docs of a different variant to understand.

Reply:
Ah, my bad, I've missed the context, sorry!